### DISCUSSION TITLE
GatsbyJS or NextJS for Joomla!4

### DISCUSSION THREAD

**Discussion by 1337180** [joomla/joomla-cms#37146] [2. Cold Features]
Please write some kind word to this debate.

**Comment by brianteeman** [comment]
neither

**Comment by 1337180** [comment]
:) so what should I learn?

**Reply by brianteeman** [reply]
whatever you want. that wasn't your question

**Reply by 1337180** [reply]
okay, so my up to date question is - is JAMStack possible in Joomla and is there any support or knowledgebase

**Reply by C-Lodder** [reply]
@1337180 Yes, it's possible. Joomla has an API guide here: https://docs.joomla.org/J4.x:Joomla_Core_APIs. You'll need to search elsewhere for your JS framework docs
